Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION=v0.4.9
```

Comment:
You don't need to bump the version, it is done when releasing a new version. Please remove bump